### ISSUE TITLE
kaj undefined in Mask class

### ISSUE THREAD

**Issue by clementpoiret** [JensOverby/MatrixCaps#1]
Hi,

First of all, I'd like to thank you for your implementation of SegCaps in PyTorch, I wanted to try this model, but as I use Python, I did not found an implementation like yours.

In the Mask class (https://github.com/JensOverby/MatrixCaps/blob/master/SegCaps-tensorflow/capsule_layers_pytorch.py) you compute something with "kaj", e.g.:
`mask = kaj*transforms.Resize([hei, wid], interpolation = Image.BICUBIC)(mask)`

But kaj is never defined. In the tf implementation, there is only the transformation, should we just remove `kaj`?

Thanks,
Clément
